src/features/collapsible-sidebar/CollapsibleSidebar.scss

```diff
@@ -155,6 +155,39 @@ $left-sidebar-z-index: 40 !default;
             width: calc(100vw - (#{$collapsible-side-bar-h-padding} * 2));
         }
 
+        .bdl-CollapsibleSidebar-logo {
+            height: fit-content;
+            margin: 0;
+
+            .bdl-CollapsibleSidebar-toggleButton {
+                display: flex;
+                align-items: center;
+                justify-content: center;
+                padding: 18px;
+                border: none;
```

Comment:
Is this okay accessibility wise if we remove this border? We might need to check if we need to remain compliant on responsive designs.